src/components/Routes.js

```diff
@@ -1,11 +1,22 @@
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { PropTypes } from 'prop-types';
+
+import SocialMediaAuth from './SocialMediaAuth';
 import Home from './Home';
+import Signup from './Signup';
 
-export const Routes = () => (
+const Routes = ({ isAuth }) => (
   <Switch>
-    <Route exact path="/" render={props => <Home {...props} />} />
+    <Route exact path="/" render={props => <Home {...props} isAuth={isAuth} />} />
+    <Route exact path="/auth" render={props => <SocialMediaAuth {...props} />} />
```

Comment:
Does all the social loading use the same route? `/auth`?

Reply:
Yes, since the action to be taken depends on the clicked social media button